fsspec/implementations/reference.py

```diff
@@ -463,18 +478,28 @@ def write(self, field, record, base_url=None, storage_options=None):
 
         fn = f"{base_url or self.out_root}/{field}/refs.{record}.parq"
         self.fs.mkdirs(f"{base_url or self.out_root}/{field}", exist_ok=True)
+
+        if self.engine == "pyarrow":
+            df_backend_kwargs = {}
```

Comment:
There are probably some good defaults here?

Reply:
Good call! I added a few, I'll see how the tests go :) 

Absolutely not a parquet expert, so we'll see.